### PR TITLE
fix: strip html tags from ics calendar

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-server/issues/8098

just strip html tags without any formatting here, fomatting will be maintained accross all other calendar, but not only here in ics.

Screenshot from 2021-07-20 08-43-20

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by MeghalBisht** [comment]
`b"BEGIN:VCALENDAR\r\nVERSION:2.0\r\nMETHOD:PUBLISH\r\nX-WR-CALDESC:Event Calendar\r\nX-WR-CALNAME:bay!\r\nBEGIN:VEVENT\r\nSUMMARY:bay!\r\nDTSTART;TZID=Asia/Calcutta;VALUE=DATE-TIME:20210814T000000\r\nDTEND;TZID=Asia/Calcutta;VALUE=DATE-TIME:20210817T170015\r\nUID:85c22d26\r\nDESCRIPTION:Lorem Ipsum\xc2\xa0is simply dummy text of the prin\r\n ting and set sheets containing Lorem Ipsum passages\\, and more recently\r\n  with desktop publishing software like Aldus PageMaker including versions \r\n of Lorem Ipsum.\r\nLOCATION:https://localhost:4200/e/85c22d26\r\nORGANIZER:Lorem Ipsum\xc2\xa0is simply dummy text of the printi\r\n ng and typesetting industry. It has survived not on\r\n ly five centuries, but also the leap into electronic typesetting, \r\n remaining essentially unchanged. It was popularised in the 1960s with the \r\n release of Letraset sheets containing Lorem Ipsum passages, and more recen\r\n tly with desktop publishing software like Aldus PageMaker including versio\r\n ns of Lorem Ipsum.\r\nURL:https://localhost:4200/e/85c22d26\r\nEND:VEVENT\r\nEND:VCALENDAR\r\n"`


\r and \n is not removed. Can this be formatted as well?

**Comment by maze-runnar** [comment]
> `b"BEGIN:VCALENDAR\r\nVERSION:2.0\r\nMETHOD:PUBLISH\r\nX-WR-CALDESC:Event Calendar\r\nX-WR-CALNAME:bay!\r\nBEGIN:VEVENT\r\nSUMMARY:bay!\r\nDTSTART;TZID=Asia/Calcutta;VALUE=DATE-TIME:20210814T000000\r\nDTEND;TZID=Asia/Calcutta;VALUE=DATE-TIME:20210817T170015\r\nUID:85c22d26\r\nDESCRIPTION:Lorem Ipsum\xc2\xa0is simply dummy text of the prin\r\n ting and set sheets containing Lorem Ipsum passages\\, and more recently\r\n  with desktop publishing software like Aldus PageMaker including versions \r\n of Lorem Ipsum.\r\nLOCATION:https://localhost:4200/e/85c22d26\r\nORGANIZER:Lorem Ipsum\xc2\xa0is simply dummy text of the printi\r\n ng and typesetting industry. It has survived not on\r\n ly five centuries, but also the leap into electronic typesetting, \r\n remaining essentially unchanged. It was popularised in the 1960s with the \r\n release of Letraset sheets containing Lorem Ipsum passages, and more recen\r\n tly with desktop publishing software like Aldus PageMaker including versio\r\n ns of Lorem Ipsum.\r\nURL:https://localhost:4200/e/85c22d26\r\nEND:VEVENT\r\nEND:VCALENDAR\r\n"`
> 
> 
> \r and \n is not removed. Can this be formatted as well?

Yes,with some regex I can remove that.

**Comment by mariobehling** [comment]
Ok, sounds like a solution. It is possible convert text with URLs to "Text (http...)"?

**Comment by maze-runnar** [comment]
> Ok, sounds like a solution. It is possible convert text with URLs to "Text (http...)"?

I am not sure, @MeghalBisht any idea?

**Comment by MeghalBisht** [comment]
> Ok, sounds like a solution. It is possible convert text with URLs to "Text (http...)"?

Not sure what you mean by "Text (http...)".
Any example ?

**Comment by mariobehling** [comment]
The user writes "[FOSSASIA](https://fossasia.org)" in the text, then please convert it to "FOSSASIA (https://fossasia.org)".

File: app/api/custom/calendars.py [hunk 1]
```diff
@@ -6,6 +6,12 @@
 from app.api.helpers.calendar.ical import to_ical
 from app.api.helpers.permissions import to_event_id
 from app.models.event import Event
+from app.models.helpers.versioning import strip_tags
```
Comment (MeghalBisht) [inline app/api/custom/calendars.py:9]:
```suggestion
```

**Review by MeghalBisht — DISMISSED** [review]

File: app/api/custom/calendars.py [hunk 2]
```diff
@@ -6,6 +6,11 @@
 from app.api.helpers.calendar.ical import to_ical
 from app.api.helpers.permissions import to_event_id
 from app.models.event import Event
+import html2text
+
```
Comment (MeghalBisht) [inline app/api/custom/calendars.py:10]:
```suggestion
```

**Review by ranjitkshah — APPROVED** [review]

File: app/api/custom/calendars.py (hunk 2 above)
Comment (MeghalBisht) [reply app/api/custom/calendars.py:10]:
```suggestion


```

File: app/api/custom/calendars.py [hunk 3]
```diff
@@ -6,6 +6,10 @@
 from app.api.helpers.calendar.ical import to_ical
 from app.api.helpers.permissions import to_event_id
 from app.models.event import Event
+import html2text
```
Comment (MeghalBisht) [inline app/api/custom/calendars.py:9]:
```suggestion
import html2text

```

**Review by MeghalBisht — DISMISSED** [review]

**Comment by maze-runnar** [comment]
Screenshot from 2021-07-27 19-23-15

**Comment by maze-runnar** [comment]
Screenshot from 2021-07-27 20-59-50

**Comment by maze-runnar** [comment]
closing due to error is not reproducible.

**Pull request closed without merging**